src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -253,21 +256,22 @@ describeEachAppLayout({ sizes: ['desktop'] }, ({ theme }) => {
 
     expect(wrapper.findSplitPanelOpenButton()!.getElement()).toBeInTheDocument();
   });
-});
 
-describe('Visual refresh only features', () => {
-  beforeEach(() => {
-    (useVisualRefresh as jest.Mock).mockReturnValue(true);
-  });
-  afterEach(() => {
-    (useVisualRefresh as jest.Mock).mockReset();
+  test(`${theme === 'refresh-toolbar' ? 'does not render' : 'renders'} the side open-button correctly when split panel is in bottom position`, () => {
+    const { wrapper } = renderComponent(
+      <AppLayout
+        splitPanel={defaultSplitPanel}
+        splitPanelOpen={true}
+        onSplitPanelToggle={noop}
+        splitPanelPreferences={{ position: 'bottom' }}
+        onSplitPanelPreferencesChange={noop}
+      />
+    );
+    expect(wrapper.findToolsToggle()).toBeTruthy();
+    expect(wrapper.findSplitPanelOpenButton())[`toBe${theme === 'refresh-toolbar' ? 'Truthy' : 'Falsy'}`]();
```

Comment:
seems unathoradox however using `toBe(theme === 'refresh-toolbar' ? true : null)` did not work.  Perhaps a different check?